src/reducers.js

```diff
@@ -165,10 +169,27 @@ function userDataReducer(state = Immutable.Map({
   }
 }
 
+function invoicePageReducer(state = Immutable.Map({
```

Comment:
Because we're not actually using the loading state, I think this part of the state should go away.